### DISCUSSION TITLE
WiFi adapter not accepting certificates

### DISCUSSION THREAD

**Discussion by michelef** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#297] [Q&A]
Hi,

As Chris showed the adpter works well in Sonoma.
My problem is that my company requires accepting a certificate before connecting to WiFi (see attached dialog) but it seems that the adapter sowftare does not know how to deal with it.
Any help/solution?
Tanks
Michele
![dialog](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/74962/685cba85-272f-4057-81e1-d8da8ddb73e5)

**Comment by chris1111** [comment]
I don't think there is a solution
